src/python/pants/engine/native.py

```diff
@@ -562,22 +563,12 @@ class Native(object):
   @staticmethod
   def create(options):
     """:param options: Any object that provides access to bootstrap option values."""
-    binary_util = BinaryUtil.Factory.create()
-    return Native(binary_util,
-                  options.native_engine_version,
-                  options.native_engine_supportdir,
-                  options.native_engine_visualize_to)
+    return Native(options.native_engine_visualize_to)
 
```

Comment:
Not a comment for you, but probably for @benjyw / @kwlzn : should we consider reverting c34fa8669fa885fc6c6a188c13fa9226ecc9d366 after this lands? Sounds like it had some fallout anyway: #4917

Reply:
no, the daemon externalization patch depends on c34fa86 and will move all of the `pantsd` and `watchman` scoped options as well as some global options into bootstrap as well. so we'll have to solve or accept #4917 very soon either way.